packages/xo-web/src/common/xo-compression/select-compression.js

```diff
@@ -0,0 +1,65 @@
+import _ from 'intl'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { injectState, provideState } from 'reaclette'
+import { omit } from 'lodash'
+
+import decorate from '../apply-decorators'
+import Select from '../form/select'
+import { connectStore } from '../utils'
+import { createGetObject } from '../selectors'
+
+import { isZstdSupported } from '.'
+
+const COMMON_OPTIONS = [
+  {
+    label: _('stateDisabled'),
+    value: '',
+  },
+  {
+    label: _('chooseCompressionGzipOption'),
+    value: 'native',
+  },
+]
+
+const ZSTD_OPTION = {
+  label: _('chooseCompressionZstdOption'),
+  value: 'zstd',
+}
+
+const SELECT_COMPRESSION_PROP_TYPES = {
+  container: PropTypes.string,
```

Comment:
`container` as a props of `SelectCompression`! 
may be instead `zstdExport: PropTypes.bool`

Reply:
I do that for optimization. But you're right. `SelectCompression` component should not do things not relative to it's principal functionality.